solution/backend/cmcs_regulations/settings/base.py

```diff
@@ -212,7 +213,7 @@
     "https://docs.google.com/forms/d/e/1FAIpQLSdcG9mfTz6Kebdni8YSacl27rIwpGy2a7GsMGO0kb_T7FSNxg/viewform?embedded=true"
 )
 
-DEPLOY_NUMBER = os.environ.get("DEPLOY_NUMBER", "0000")
+DEPLOY_NUMBER = os.environ.get("DEPLOY_NUMBER", datetime.now())
```

Comment:
changing the default to `datetime.now` so when working locally, the assets are fetched every refresh